jumanji/environments/routing/robot_warehouse/env.py

```diff
@@ -271,6 +265,12 @@ def update_state_scan(
             update_state_scan, (grid, agents, shelves, 0), actions
         )
 
+        # check for agent collisions
+        collisions = jax.vmap(functools.partial(utils.is_collision, grid))(
+            agents, self.agent_ids
+        )
```

Comment:
Up to you, but I prefer

```suggestion
        collisions = jax.vmap(utils.is_collision, (None, 0, 0))(
            grid, agents, self.agent_ids
        )
```

Reply:
I'm agnostic to this, up to you.

Reply:
Keeping as is.